src/groups/bmq/bmqsys/bmqsys_statmonitor.cpp

```diff
@@ -270,25 +270,26 @@ void StatMonitor::snapshot()
     // Convenient macro to retrieve the value 'T' applying a multiplier 'M'
 #define PERFVAL(T, M)                                                         \
     (static_cast<bsls::Types::Int64>(                                         \
-        it->latestValue(balb::PerformanceMonitor::BAEA_##T) * M))
+        it->latestValue(balb::PerformanceMonitor::T) * M))
 
     // CPU: report CPU values * k_STAT_MULTIPLIER because statContext only
     // handles Int64, not float, so multiply by k_STAT_MULTIPLIER to keep some
     // decimal digits, and we'll divide by k_STAT_MULTIPLIER before reporting
     // the stats
     d_cpuStatContext_mp->reportValue(k_STAT_CPU_SYSTEM,
-                                     PERFVAL(CPU_UTIL_SYSTEM,
+                                     PERFVAL(e_CPU_UTIL_SYSTEM,
```

Comment:
Keeping `e_` together with the rest of the name allows easy searching for these enums in the codebase if necessary